.travis.yml

```diff
@@ -13,4 +13,5 @@ install:
   - make init
 script:
   - make build test lint coverage
+  - make docker-all
```

Comment:
so where did we land on this one? Keep it as it is in the PR?

re: building two ways... from previous threads there appears to be some people who want to build via docker and some who want to build natively.  Until the time when one camp is kicked off the island and we only have one way to build, I think the CI needs to build both - otherwise we could merge something that breaks the "other" build path.  btw - I would prefer to have just one way - docker.  :-)